xbmc/guilib/GUIInfoTypes.h

```diff
@@ -125,6 +126,10 @@ class CGUIInfoLabel
    */
   static std::string ReplaceAddonStrings(const std::string &label);
 
+  typedef boost::function<std::string(const std::string&)> StringReplacerFunc;
+  static bool ReplaceDollarString(const std::string &strInput, std::string &strOutput, const std::string &str, const StringReplacerFunc &func);
+  static bool ReplaceDollarString(std::string &work, const std::string &str, const StringReplacerFunc &func);
```

Comment:
Could we get some documentation for these two methods (to also be able to tell the difference)?
And please re-order the parameters so that the input parameters come first and the output parameters last.